include/Model/Languages.php

```diff
@@ -734,6 +734,33 @@ public function clean_cache(): void {
 		$this->cache->clean();
 	}
 
+	/**
+	 * Deletes the transient from the options table since WordPress does not do it when using object cache.
+	 *
+	 * @since 3.8
+	 *
+	 * @return void
+	 */
+	public function delete_transient_from_options_table(): void {
+		if ( wp_using_ext_object_cache() || wp_installing() ) {
```

Comment:
Why `wp_installing()` too?

Reply:
As discussed during our meeting, this is the way WordPress "prevent" transient from being deleted in DB. So we have to go against in any cases.